src/Effect/ReadFS.hs

```diff
@@ -107,7 +119,30 @@ deriving instance Ord (ReadFSF a)
 instance ToDiagnostic ReadFSErr where
   renderDiagnostic = \case
     FileReadError path err -> "Error reading file " <> pretty path <> ":" <> line <> indent 4 (pretty err)
-    FileParseError path err -> "Error parsing file " <> pretty path <> ":" <> line <> indent 4 (pretty err)
+    FileParseError path err PythonRequirementErrFmt ->
+      vsep
+        [ "Error parsing file: " <> pretty path <> "."
+        , ""
+        , "We occasionally find files we think are python requirements.txt files but"
+        , "aren't. If this file isn't a python requirements.txt file, this error can"
+        , "be safely ignored."
+        , ""
+        , "Details:"
+        , indent 4 (pretty err)
+        , ""
+        , "If you believe this to be a defect - please report bug with copy of the file"
+        , pretty $ "to FOSSA support: " <> supportUrl
+        ]
+    FileParseError path err DefaultParseErrMsgFmt ->
+      vsep
+        [ "Error parsing file: " <> pretty path <> "."
+        , ""
+        , "Details:"
+        , indent 4 (pretty err)
+        , ""
+        , "If you believe this to be a defect - please report bug with copy of the file"
+        , pretty $ "to FOSSA support: " <> supportUrl
+        ]
```

Comment:
It doesn't have to be now, but this should eventually be split into a new module.

Reply:
- [ ] Breakout `renderDiag` method for parserError into its own module.